src/Fixer/ClassNotation/NullPropertyDeclarationFixer.php

```diff
@@ -66,10 +66,14 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 $index = $tokens->getNextMeaningfulToken($index);
 
                 if ($tokens[$index]->equals('=')) {
-                    $valueTokenIndex = $index = $tokens->getNextMeaningfulToken($index);
+                    $index = $tokens->getNextMeaningfulToken($index);
 
                     if ($tokens[$index]->equals([T_STRING, 'null'], false)) {
-                        $tokens->clearRange($varTokenIndex + 1, $valueTokenIndex);
+                        for ($i = $varTokenIndex + 1; $i <= $index; ++$i) {
+                            if (!$tokens[$i]->isComment() && !($tokens[$i]->isWhitespace() && false !== strpos($tokens[$i]->getContent(), "\n"))) {
```

Comment:
:+1: for the first (can you do it?),
second not so much, we have better fixers to deal with line breaks